ui.frontend/src/main/webpack/site/js/components/searchResultsTab.ts

```diff
@@ -10,9 +9,6 @@ type BuildSearchResults = {
   searchForm: HTMLFormElement
   searchFormParent: HTMLElement | null
   loadMoreButtonText: string
-  onSearchItemClick?: OnSearchItemClickCallback
-  onSwitchTab?: CallbackFunction
-  onLoadMoreButtonClick?: CallbackFunction
```

Comment:
**Thought**

@igorsimovski in your PR didn't you use the onSearchItemClick? Apparently, we are removing all callbacks in favor of the dispatch custom event. We must be sure we do not send or expect any callback to be defined/used.